kernel/build.rs

```diff
@@ -6,11 +6,12 @@ where
     F: FnMut(&mut Command) -> &mut Command,
 {
     let mut command = Command::new(name);
-    println!("Running {:?}", command);
+    println!("Running {command:?}");
     let configured = configure(&mut command);
-    if configured.status().is_err() {
-        panic!("failed to execute {:?}", configured);
-    }
+    assert!(
```

Comment:
Readability change. See this [clippy lint](https://rust-lang.github.io/rust-clippy/master/index.html#/manual_assert). Applies to all if-panic in the codebase.